File: SantanderTeste/app/src/main/java/com/example/alessandrofsouza/santanderapp/presentation/pages/contact/ContactContract.java
package com.example.alessandrofsouza.santanderapp.presentation.pages.contact;

import com.example.alessandrofsouza.santanderapp.domain.model.Cell;
import com.example.alessandrofsouza.santanderapp.presentation.pages.base.BasePresenter;
import com.example.alessandrofsouza.santanderapp.presentation.pages.base.BaseView;

import java.util.ArrayList;

public interface ContactContract {

    interface View extends BaseView<Presenter> {
        void showContactCells(ArrayList<Cell> cellList);
    }

    interface Presenter extends BasePresenter {
        void getContactCells();
    }

}

File: source/app/src/main/java/br/com/andreyneto/testesantander/ui/contact/ContactContract.kt
package br.com.andreyneto.testesantander.ui.contact

import android.content.Context
import br.com.andreyneto.testesantander.model.Cell
import br.com.andreyneto.testesantander.model.Info
import br.com.andreyneto.testesantander.model.MoreInfo
import br.com.andreyneto.testesantander.model.Screen
import br.com.andreyneto.testesantander.ui.base.BasePresenter
import br.com.andreyneto.testesantander.ui.base.BaseView

class ContactContract {

    interface View: BaseView<Presenter> {
        fun showCells(cells: List<Cell>)
        fun showForm()
        fun formSended()
    }

    interface Presenter: BasePresenter {
        fun getCells()
        fun sendForm()
        fun newMessage()
    }
}